commands/system/toggle-airpods.template.swift

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/swift
+
+// Required parameters:
+// @raycast.schemaVersion 1
+// @raycast.title Toggle AirPods
+// @raycast.mode silent
+// @raycast.packageName Audio
+//
+// Optional parameters:
+// @raycast.icon images/airpod.png
+//
+// Documentation:
+// @raycast.description Toggle AirPods bluetooth device
+// @raycast.author Nichlas W. Andersen
+// @raycast.authorURL https://github.com/itsnwa
+
+import IOBluetooth
+
+func toggleAirPods() {
+    guard let bluetoothDevice = IOBluetoothDevice(addressString: "Your AirPods MAC address") else {
+        print("Device not found")
+        exit(-2)
```

Comment:
@itsnwa here you can use `exit(1)` (fail) and `exit(0)` (success), this is a pattern in the CLI world! ;)
```suggestion
        exit(1)
```